client/src/legacy/LeftAndMain.EditForm.js

```diff
@@ -96,9 +98,6 @@ $.entwine('ss', function($){
         }
       }
 
-      // Reset error display
-      this.setValidationErrorShown(false);
```

Comment:
We can't rely on this being reset BEFORE we fire a validation message, since `onadd` is fired asyncronously to the `redrawTabs()` call which triggers a call to `onafterredrawtabs()` which relies on this value.